packages/mongo-nav/src/types.ts

```diff
@@ -100,11 +100,11 @@ export interface CurrentOrganizationInterface extends OrganizationInterface {
 }
 
 export interface DataInterface {
-  readonly account: AccountInterface;
-  currentOrganization: CurrentOrganizationInterface;
-  currentProject: CurrentProjectInterface;
-  readonly organizations: Array<OrganizationInterface>;
-  readonly projects: Array<ProjectInterface>;
+  readonly account?: AccountInterface;
```

Comment:
Do we really need to make all these properties optional? What's the impetus for this?

Really the data itself is optional, but we should be able to safely assume certain things of the structure of the data itself.

Namely that account will always exist, currentOrg and currentProject can definitely be undefined, but I believe organizations and projects will always at least be empty arrays.

Reply:
I think was just trying to defend against 'what ifs...' and handling cases where certain pieces of data are missing (even though they probably shouldn't be). I'll change account to be non-optional, same with orgs and projects